src/components/events/commonEventFields/eventLists.tsx

```diff
@@ -1,23 +1,23 @@
 import { Typography } from '@mui/material';
-import { eventKeyDisplay, eventLists } from '../../../types/configurations';
+import { EVENT_KEY_DISPLAY, EVENT_LISTS } from '../../../types/configurations';
 import { IEvent } from '../../../types/interfaces';
 import ListGenerator from '../../general/listGenerator';
 import List from '../../general/list';
 
-interface ListsProps {
+interface Props {
     event: IEvent;
     editable: boolean;
     setAttr: (attr: keyof IEvent, value: any) => void;
```

Comment:
not needed at al but
if you want values of a type then
```
export type SomeTypeKeys = keyof SomeType;
export type SomeTypeKeysValues = SomeType[SomeTypeKeys ];
```

it will create a ty;e literal taht contaions all possible values (string | string[] | Date[] | {
    weapons: WeaponConfig[];
    versions: VersionConfig[];
} | undefined )

Reply:
Done